tide/master/MasterApplication.cpp

```diff
@@ -73,6 +73,11 @@
 
 #include <stdexcept>
 
+namespace
+{
+const int MOUSE_MARKER_ID = 123456;
```

Comment:
yes, it can be any value, as long as it does not conflict with the touch point ids. How would you do it?